### DISCUSSION TITLE
Can you explain how to use session id and tokens ?

### DISCUSSION THREAD

**Discussion by brightchip** [accounts-js/accounts#1144] [Q&A] [answered]
I made a success login with the Mutation 
` authenticate(serviceName:"password",params:{password:"2111" ,user:{username:"test"}}){
  sessionId
  tokens{
    refreshToken
    accessToken
  }
  user{
    username
    displayName
  }
}
}`,
 which returns sessionId , tokens and user info.

I'm wondering how do I resume login ?  Like auto login the user with the tokens and session id after user logged in before

What is the "access_token_secret" ? 
![image](https://user-images.githubusercontent.com/28724907/119300905-25c85600-bc94-11eb-89a8-d2beda4b57f4.png)

How do I disable some mutations like:  impersonate ?
How to use the mutaation : "getUser" as there's no input field.

**Comment by pradel** [comment, marked as answer]
- `access_token_secret` is only used by the oauth package where you need to send an access token and an access token secret coming from twitter for example
- "How do I disable some mutations like: impersonate ?" - it's not enabled by default so if a user try to call it, it will never work.
- "I'm wondering how do I resume login ? Like auto login the user with the tokens and session id after user logged in before" - you need to use the returned access token and pass it as a value for an `Authorisation` header along with your request

**Reply by brightchip** [reply]
Thank you for the explanation , I just got everything works
